scripts/whitelist-fiscal-code-script/README.md

```diff
@@ -0,0 +1,52 @@
+# Whitelist Fiscal Code Script
+
+This script is used to whitelist fiscal codes in the database.
+
+### Prerequisites
+
+- NodeJS ^22.15.0
+- NPM ^10.9.2
+
+### Quickstart
+
+**STEP 1**
+Make sure there is a CSV file under the `fiscalCodes/` directory. This directory is used to store the fiscal codes to be whitelisted and the file name must be `fiscalcodes.csv`. This CSV file must have a column named `fiscalCode` and the values must be the fiscal codes to be whitelisted, like this:
+
```

Comment:
If the file is unique maybe it doesn't make sense to have a folder too, what do you think?